app/routes/_gcn.user.email/email_notices.server.ts

```diff
@@ -216,3 +218,25 @@ export async function sendTestEmail(to: string) {
     body: 'This is a test message from GCN Notices.',
   })
 }
+
+export async function sendNewsAnnouncementEmail(
+  subject: string,
+  body: string,
+  user?: User
+) {
+  if (!user || !user.groups.includes(moderatorGroup))
+    throw new Response(null, { status: 403 })
+
+  const db = await tables()
+  const to = (
+    await db.announcement_subscriptions.scan({
+      ProjectionExpression: 'email',
+    })
+  ).Items.map((item) => item.email)
```

Comment:
You'll definitely have to paginate this. We have enough users that they won't all be returned in a single scan call. See, for example, https://github.com/nasa-gcn/gcn.nasa.gov/blob/main/app/table-streams/circulars/index.ts#L57-L65.